org.mythtv.MythTV.yaml

```diff
@@ -0,0 +1,1063 @@
+id: org.mythtv.MythTV
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: mythfrontend
+finish-args:
+  - --device=all
+  - --persist=.mythtv
+  - --filesystem=home/.mythtv
```

Comment:
why do you need both?

Reply:
> why do you need both?

Otherwise it can't initially create the directory on its own

Reply:
this currently doesn't make sense it's either persist or the other one.

Reply:
> > why do you need both?
> 
> Otherwise it can't initially create the directory on its own

I doubt. These lead to two different directories. You should use `--persist`.